src/css/flexboxgrid.css

```diff
@@ -224,6 +224,13 @@
     width: var(--container-sm, 46rem);
   }
 
+  [class*="col-sm"] {
```

Comment:
speaking of reducing bundle size, why you cannot to put styles in one place and separate attributes with `,`?
```
[class*="col-xs"],
[class*="col-sm"],
[class*="col-md"],
[class*="col-lg"] {
    box-sizing: border-box;
    flex: 0 0 auto;
    padding-right: var(--half-gutter-width, 0.5rem);
    padding-left: var(--half-gutter-width, 0.5rem);
}
```

Reply:
I don't know which one is preferred though, so I see both ones as valid.